src/views/AllCharacters/AllCharacters.test.jsx

```diff
@@ -0,0 +1,40 @@
+import { render, screen, waitForElementToBeRemoved } from '@testing-library/react';
+import AllCharacters from './AllCharacters';
+
+test('header and dropdown renders in a loading state', async () => {
+  render(<AllCharacters />);
+
+  const header = screen.getByRole('heading', {
+    name: /welcome to hogwarts/i,
+  });
+  const crest = screen.getByAltText(/hogwarts crest/i);
+  const loading = screen.getByText(/loading characters/i);
+
+  expect(loading).toBeInTheDocument();
+  expect(header).toBeInTheDocument();
+  expect(crest).toBeInTheDocument();
+  await waitForElementToBeRemoved(loading);
```

Comment:
Better practice would be to pass line 11 in as a callback to the `waitForElementToBeRemoved`